x-pack/plugins/security_solution/public/common/components/cell_actions/index.tsx

```diff
@@ -61,7 +61,6 @@ export const SecurityCellActions: React.FC<SecurityCellActionsProps> = ({
   // Make a dependency key to prevent unnecessary re-renders when data object is defined inline
   // It is necessary because the data object is an array or an object and useMemo would always re-render
   const dependencyKey = JSON.stringify(data);
-
```

Comment:
not a big deal, but this change could be undone so no modifications would happen in this file